internal/repository/storage_repo.go

```diff
@@ -0,0 +1,59 @@
+package repository
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/Melikhov-p/url-minimise/internal/config"
+	"github.com/Melikhov-p/url-minimise/internal/models"
+	"github.com/Melikhov-p/url-minimise/internal/storage"
+)
+
+type IStorage interface {
```

Comment:
Как мы обсуждали, в Go интерфйсы не принято называть начиная с "I"

```suggestion
type Storage interface {
```